components/FloatingActionButton/FloatingActionButton.jsx

```diff
@@ -9,7 +9,13 @@ const FloatingActionButton = () => {
     return (
         show && (
             <div className={`${styles.fabWrapper} flex-row-center`}>
-                <Link href='#'>
+                <Link
```

Comment:
I just realized this component should not use `<Link />` it's just an anchor to the top of the page. It's not navigating to anywhere. You could just remove the next link and use a regular `<a />` targeting '#'. This will solve the issue and make the component less expensive

Reply:
Changed